### ISSUE TITLE
Cant buy whistle

### ISSUE THREAD

**Issue by carlacolon3** [NoteBoxz/LethalMin#14]
cant buy the whistle. cant find it cant put it in the console nothin, and idk what I'm doing wrong or if its a bug or what

**Comment by NoteBoxz** [comment]
Could you send me your mod-list or your r2modman profile code?

**Comment by carlacolon3** [comment]
yeah sure, will this suffice? unsure if there's a better way you meant or smthn

![Screenshot 2024-11-17 170906](https://github.com/user-attachments/assets/6852ba0a-f605-4dbf-99df-84f291e9aea1)

**Comment by carlacolon3** [comment]
hello? im unsure how to like, get your attention in this. wondering if there's any idea of what is wrong
